src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,43 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const changeTab = (selectedId, tab) => {
+    if (selectedId !== tab.id) {
+      onTabSelected(tab);
+    }
+  };
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(el => (
```

Comment:
Don't cut variable naming. Use `element` or `tab`. Fix everywhere
```suggestion
          {tabs.map(tab => (
```